rtflib/tests/test_rtflib.py

```diff
@@ -75,5 +75,38 @@ def test_table_borders(self):
         with open(os.path.join(self.path, "table_widths_borders.rtf")) as fh:
             self.assertEqual(rtf.rtf_code.strip(), fh.read())
 
+    def test_table_borders(self):
+        rtf = Rtf()
+        page_orientation = RtfCode(PAGE_LAYOUTS["A4_landscape"])
+        rtf.preelements.append(page_orientation)
+        rtf.add(Line("hello world. A Line of text.\n"))
+        rtf.add(Line("hello world. A Line of text.\n", format=Format(bold=True,strike=True,size=16)))
+        rtf.add(Line("hello world. A Red line\n", color=Color(255, 0, 0)))
```

Comment:
See other comment. I propose we change the API so that the user should register the color here with `rtf.add_color(..)`. Then each `Color` object will store a color ID internally

Reply:
do you mean the user would do something like:
```
rtf = Rtf()
myred = rtf.add_color(255, 0, 0)
  rtf.add(Line("hello world. A Red line\n", color=myred))
```
so myred would be a Color()